code/modules/urist/machinery/uristvending.dm

```diff
@@ -175,3 +175,70 @@ Please keep it tidy, by which I mean put comments describing the item before the
 	products = list(/obj/item/weapon/storage/belt/vanity/leather = 5,/obj/item/weapon/storage/belt/vanity/cowboy = 5,/obj/item/weapon/storage/belt/vanity/black = 10,/obj/item/weapon/storage/belt/vanity/red = 10,/obj/item/weapon/storage/belt/vanity/green = 10,/obj/item/weapon/storage/belt/vanity/purple = 10,/obj/item/weapon/storage/belt/vanity/blue = 10,/obj/item/weapon/storage/belt/vanity/orange = 10)
 	prices = list(/obj/item/weapon/storage/belt/vanity/leather = 250,/obj/item/weapon/storage/belt/vanity/cowboy = 250,/obj/item/weapon/storage/belt/vanity/black = 100,/obj/item/weapon/storage/belt/vanity/red = 100,/obj/item/weapon/storage/belt/vanity/green = 100,/obj/item/weapon/storage/belt/vanity/purple = 100,/obj/item/weapon/storage/belt/vanity/blue = 100,/obj/item/weapon/storage/belt/vanity/orange = 100)
 	contraband = list(/obj/item/weapon/storage/belt/utility = 1)
+
+//Assault mode ammo vender
+
+/obj/machinery/vending/urist/assaultammodispenser
+	name = "ANFOR Weapon Cache"
+	desc = "An automated ANFOR supply cache for efficent storage and distribution of weapons, ammunition and material."
+	icon_state = "clothing2"
+	vend_reply = "Git some!"
+	product_ads = "Food for your rifle.;Out of rounds? We're here for you.;Resupply here!"
+	product_slogans = "Bullets for every occasion!;Tactical reloads!;RAMIREZ!"
+	vend_delay = 5
+	products = list(
+		//ammo
+		/obj/item/ammo_magazine/a556/a22 = 60,
+		/obj/item/ammo_magazine/a762/a18 = 60,
+		/obj/item/ammo_magazine/a9mm = 60,
+		/obj/item/weapon/storage/box/shotgunammo = 60,
+		/obj/item/weapon/storage/box/shotgunshells = 60,
+		/obj/item/ammo_magazine/c45m/a7 = 60,
+		//guns
+		/obj/item/weapon/gun/projectile/automatic/a22 = 10,
+		/obj/item/weapon/gun/projectile/a18 = 10,
+		/obj/item/weapon/gun/projectile/automatic/asmg = 10,
+		/obj/item/weapon/gun/projectile/shotgun/pump/combat/A41 = 10,
+		/obj/item/weapon/gun/projectile/colt/a7 = 20,
+		//attachments
+		/obj/item/weapon/gunattachment/grenadelauncher = 10,
+		/obj/item/weapon/gunattachment/scope/a18 = 10,
+		//gernades and mines
+		/obj/item/weapon/storage/box/anforgrenade = 10,
+		/obj/item/weapon/storage/box/mines = 3
+		)
+	contraband = list(
+		/obj/item/weapon/storage/box/flashshells = 5,
+		/obj/item/weapon/storage/box/beanbags = 5,
+		/obj/item/ammo_magazine/c38 = 10,
+		/obj/item/ammo_magazine/c45m = 10,
+		/obj/item/ammo_magazine/a556 = 10,
+		/obj/item/ammo_magazine/c762 = 10
+		)
+		
+//Assault mode clothing and armor vender
+		
+/obj/machinery/vending/urist/assaultclothingdispenser
+	name = "ANFOR Equipment Cache"
+	desc = "An automated ANFOR supply cache for efficent storage and distribution of armor and equipment."
+	product_slogans = "Look like a real soldier."
+	product_ads = "Keep yourself covered.;Protection for our fighting forces."
+	vend_delay = 15
+	vend_reply = "Suit up!"
+	icon_state = "clothing2"
+	products = list(
+		/obj/item/clothing/suit/urist/armor/anfor/nco = 2,
```

Comment:
remove the specialized gear from here, just use the basic marine stuff. so this, the engi and med stuff and the softcap, and drop the rest of the stuff to 20